src/main/java/Deadline.java

```diff
@@ -0,0 +1,64 @@
+import java.time.LocalDate;
+import java.time.LocalTime;
+
+public class Deadline extends Task {
+    private final LocalDate DeadlineDate;
+    private final LocalTime DeadlineTime;
+
+    /**
+     * Constructor of Deadline class.
+     *
+     * @param name name of deadline.
+     * @param isDone deadline status.
+     * @param DeadlineDate date of deadline
+     * @param DeadlineTime time of deadline
```

Comment:
Javadoc comments should start with caps :)
e.g @param x  X coordinate of position. Instead of @param x  x coordinate of position.